doc/SupportedC++Features.md

```diff
@@ -26,16 +26,17 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 
 OS      | Architecture | Build Compiler | Version
 --------|--------------|----------------|--------
-Linux   | x86          | g++            | [4.8](https://gcc.gnu.org/gcc-4.8/cxx0x_status.html)
-Linux   | s390x        | g++            | [4.8](https://gcc.gnu.org/gcc-4.8/cxx0x_status.html)
-Linux   | ppc64le      | XLC            | [12.1](https://www.ibm.com/developerworks/community/blogs/5894415f-be62-4bc0-81c5-3956e82276f3/entry/xlc_compiler_s_c_11_support50?lang=en)
-Linux   | ppc64le      | g++            | [4.8](https://gcc.gnu.org/gcc-4.8/cxx0x_status.html)
-AIX     | ppc64        | XLC            | [12.1](https://www.ibm.com/developerworks/community/blogs/5894415f-be62-4bc0-81c5-3956e82276f3/entry/xlc_compiler_s_c_11_support50?lang=en)
-z/OS    | s390x        | XLC            | [v2r2](http://www-01.ibm.com/support/docview.wss?uid=swg27036892)
-Windows | x86-64       | MSVC           | [2010 (version 10)](https://docs.microsoft.com/en-us/previous-versions/hh567368(v=vs.140))
-
-* Note: moving to gcc 7.3
-* Note: moving to msvc 2017
+Linux   | ARMv7        | g++            | [7.3](https://gcc.gnu.org/onlinedocs/7.3.0/)
+Linux   | AArch64      | g++            | [7.3](https://gcc.gnu.org/onlinedocs/7.3.0/)
+Linux   | x86/x86-64   | g++            | [7.3](https://gcc.gnu.org/onlinedocs/7.3.0/)
+Linux   | s390x        | g++            | [7.3](https://gcc.gnu.org/onlinedocs/7.3.0/)
+Linux   | ppc64le      | g++            | [7.3](https://gcc.gnu.org/onlinedocs/7.3.0/)
+Linux   | ppc64le      | XLC            | [12.1](https://www.ibm.com/support/pages/system/files/support/swg/swgdocs.nsf/0/330533a7bf467181852579ad001c960f/$FILE/compiler.pdf)
+AIX     | ppc64        | XLC            | [12.1](https://www.ibm.com/support/pages/system/files/support/swg/swgdocs.nsf/0/330533a7bf467181852579ad001c960f/$FILE/compiler.pdf)
```

Comment:
Are these versions for Linux and AIX XLC compilers correct? From the [conversation](https://github.com/eclipse/omr/issues/7359#issuecomment-2161689154) in the issue, I think 13.1.3 is the minimum version.

Reply:
Yes, you're right.  Thanks for spotting that.